### PR TITLE
Create text-pullrequest

### PR DESCRIPTION
**Pull request by sadioci2**
add of docker-compose file on text-pullrequest

### CODE REVIEW

File: text-pullrequest [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    image: nginx:latest
+    ports:
+      - "80:80"
+    volumes:
+      - ./frontend:/usr/share/nginx/html
+    networks:
+      - my_network
+
+  backend:
+    image: node:latest
```
Comment (s7maryam) [inline text-pullrequest:14]: ok

Comment (sadioci2) [reply text-pullrequest:14]: thank you maryam :)

**Review by s7eric — APPROVED** [review]

File: text-pullrequest (hunk 1 above)
Comment (s7maryam) [reply text-pullrequest:14]: uwc

**Review by Djurizt — APPROVED** [review]

**Review by sandargo — COMMENTED** [review]
the file looks good now

**Review by sandargo — COMMENTED** [review]
the changes are in order

**Pull request merged**